### PR TITLE
Update Assistants Creation UI

### PR DESCRIPTION
**Pull request by pablodanswer**
New assistants creation page UI

- Reorganization
- Padding
- Icons for dropdowns
- New dropdowns / layout components

### CODE REVIEW

**Comment by pablodanswer** [comment]
Screenshot 2024-06-26 at 11 27 57 AM

File: web/src/components/admin/connectors/Field.tsx [hunk 1]
```diff
@@ -20,16 +26,67 @@ export function SectionHeader({
...
-  return <div className="block font-medium text-base">{children}</div>;
+export function Label({
+  children,
+  small,
+}: {
+  children: string | JSX.Element;
+  small?: boolean;
+}) {
+  return (
+    <div className={`block font-medium base ${small ? "text-sm" : "tex-base"}`}>
```
Comment (Weves) [inline web/src/components/admin/connectors/Field.tsx]: `tex-base` -> `text-base`

File: web/src/components/admin/connectors/Field.tsx [hunk 2]
```diff
@@ -20,16 +26,67 @@ export function SectionHeader({
...
+  children: string | JSX.Element;
+}) {
+  return (
+    <TooltipProvider delayDuration={50}>
+      <Tooltip>
+        <TooltipTrigger>
+          <FiInfo size={12} />
+        </TooltipTrigger>
+        <TooltipContent side="top" align="center">
+          <p className="bg-neutral-900 max-w-[200px] mb-1 text-sm rounded-lg p-1.5 text-white">
```
Comment (Weves) [inline web/src/components/admin/connectors/Field.tsx]: prefer using a named tailwind color here (both for neutral-900 and for white)

File: web/src/components/admin/connectors/Field.tsx [hunk 3]
```diff
@@ -62,22 +123,44 @@ export function TextFormField({
...
+        {tooltip && <ToolTipDetails>{tooltip}</ToolTipDetails>}
+
+        {error ? (
+          <ManualErrorMessage>{error}</ManualErrorMessage>
+        ) : (
+          !hideError && (
+            <ErrorMessage
+              name={name}
+              component="div"
+              className="text-red-500 my-auto text-sm"
```
Comment (Weves) [inline web/src/components/admin/connectors/Field.tsx]: `text-red-500` -> `text-error`

File: web/src/components/admin/connectors/Field.tsx [hunk 4]
```diff
@@ -115,29 +190,34 @@ interface BooleanFormFieldProps {
...
+  alignTop,
 }: BooleanFormFieldProps) => {
   return (
     <div className="mb-4">
       <label className="flex text-sm">
         <Field
           name={name}
           type="checkbox"
-          className="mx-3 px-5 w-3.5 h-3.5 my-auto"
+          className={`${noPadding ? "mr-3" : "mx-3"}  px-5 w-3.5 h-3.5 ${alignTop ? "mt-1" : "my-auto"}`}
```
Comment (Weves) [inline web/src/components/admin/connectors/Field.tsx]: nit: remove extra space

**Review by Weves — CHANGES_REQUESTED** [review]
Overall looks awesome! Left a couple small style comments. Two other quick things:

- Build is failing, seems to be an incorrect path somewhere?
- This may have already changed, but I slightly prefer the form being centered simply because this is how the other forms behave. If we want to move them to the left side, we can definitely have that discussion (I'm totally open to it), but for now I'd prefer to keep them consistent

**Review by Weves — APPROVED** [review]

**Pull request merged**